bls/src/keypair.rs

```diff
@@ -229,4 +359,19 @@ mod tests {
         assert_eq!(keypair.secret, secret);
         assert_eq!(keypair.public, public);
     }
+
+    #[test]
+    #[cfg(feature = "fs")]
+    fn test_keypair_file() {
+        let out_dir = std::env::var("FARF_DIR").unwrap_or_else(|_| "farf".to_string());
+        let outfile = format!("{}/tmp/test_write_keypair_file.json", out_dir);
+        let original_keypair = Keypair::new();
+        original_keypair.write_json_file(&outfile).unwrap();
+        assert!(Path::new(&outfile).exists());
+
+        let read_keypair = Keypair::read_json_file(&outfile).unwrap();
+        assert_eq!(original_keypair, read_keypair);
+
+        fs::remove_file(&outfile).unwrap();
+    }
```

Comment:
For this test to run by default, you'll need to add this crate as a `dev-dependency` and enable the `fs` feature. Example at https://github.com/anza-xyz/solana-sdk/blob/c39b21fa43d142fe129e57593a150853df87fa9a/vote-interface/Cargo.toml#L46